src/sentry/tagstore/snuba/backend.py

```diff
@@ -523,18 +531,65 @@ def get_perf_group_list_tag_value(
     def get_generic_group_list_tag_value(
         self, project_ids, group_id_list, environment_ids, key, value
     ):
-        return self.__get_group_list_tag_value(
-            project_ids,
+        translated_params = self._translate_filter_keys(project_ids, group_id_list, environment_ids)
+        organization_id = get_organization_id_from_project_ids(project_ids)
+        start, end = _prepare_start_end(
+            datetime(2008, 5, 8),
```

Comment:
Would we maybe be better off allowing `start` to be passed as `None` and handling that case in `_prepare_start_end`? Could just get `now() - <org retention date>` to get the minimum date

Reply:
That's a good suggestion. Not too sure about the `now() - <org retention date>` though cause `_prepare_start_end` is used in the `_prepare_query_params` which is used by `bulk_raw_query` for legacy querying and that might mess up older queries.